### PR TITLE
refactor: Use hardcoding of `last_hidden_state` only if multiple output nodes exist

### LINKED ISSUE #81: Unable to use ONNX model exported by Optimum
I wish to use model on huggingface that exist as PyTorch.bin files, however `fastembed-rs` does not support these if using [optimum to convert these](https://huggingface.co/docs/optimum/en/exporters/onnx/usage_guides/export_a_model).

Optimum tends to export the last layer as `logits` and not `last_hidden_state`: but at present fastembed-rs is hardcoded to use the latter.

Desired outcome is to be able to use optimum generated ONNX models to use this tool, allowing many more models: e.g. [BioBERT](https://huggingface.co/dmis-lab/biobert-base-cased-v1.2).

### PR DESCRIPTION
**Pull request by lawrenceadams**
This PR changes the behaviour of the `get_model_info` function to try to use the only output node available. If there are multiple, then default to `last_hidden_state` as per previous behaviour.

Allows easier use of the library with ONNX models from disparate sources.

Resolves #81 

## Summary by CodeRabbit

- **Bug Fixes**
  - Improved handling of multiple output keys in text embeddings, defaulting to `last_hidden_state` for better consistency and accuracy.

### CODE REVIEW

**Review by Anush008 — APPROVED** [review]
Thank you @lawrenceadams. LGTM.
